pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -384,8 +385,7 @@ First In, First Out: Files will be uploaded in the order they were queued.</prop
               <object class="GtkRadioButton" id="use_upload_slots_radio">
                 <property name="visible">True</property>
                 <property name="hexpand">True</property>
-                <property name="label" translatable="yes">Limit number of upload slots to:</property>
-                <property name="tooltip-text" translatable="yes">If disabled, slots will automatically be determined by available bandwidth limitations.</property>
+                <property name="label" translatable="yes">Maximum number of active uploads:</property>
```

Comment:
I like the original "Fixed number of upload slots" proposal more. It's easier to understand if both options use the same terminology.